cmd/thanos/receive.go

```diff
@@ -868,6 +870,8 @@ func (rc *receiveConfig) registerFlag(cmd extkingpin.FlagClause) {
 
 	rc.forwardTimeout = extkingpin.ModelDuration(cmd.Flag("receive-forward-timeout", "Timeout for each forward request.").Default("5s").Hidden())
 
+	rc.maxBackoff = extkingpin.ModelDuration(cmd.Flag("receive-forward-max-backoff", "Maximum backoff for each forward fan-out request").Default("30s").Hidden())
```

Comment:
I think it's okay to unhide this and be more transparent about what the receiver is doing. Many people (including myself until recently) don't know there is a backoff and are often caught off guard by elevated error rates during rollouts.

Reply:
Fair point